src/status_im/utils/events_buffer.cljs

```diff
@@ -3,10 +3,21 @@
             [re-frame.core :as re-frame])
   (:require-macros [cljs.core.async.macros :refer [go-loop]]))
 
-(defonce ^:private buffer (async/chan (async/sliding-buffer 1024)))
-
+;; NOTE:(dmitryn) Using buffer here because using channel
+;; raises when there are more than 1024 pending puts.
+;; Ideally we should not exceed current buffer size.
+;; https://github.com/clojure/core.async/blob/0498ba69fda5d930e3e7568cc90ca933c19c42ca/src/main/clojure/cljs/core/async/impl/channels.cljs#L86
+(defonce ^:private buffer (async/chan (async/buffer 10000)))
```

Comment:
Just a small hint, the above can be more succinctly written as `(async/chan 1000)` - whenever you specify buffer size as an integer, fixed buffer is created by default.